health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AssumptionsList.js

```
@@ -7,10 +7,22 @@ import { useHistory } from 'react-router-dom';
 const AssumptionsList = ({ customProps, setupCompleted }) => {
     const { t } = useTranslation();
     const history = useHistory();
-
+    const campaign_dist=customProps?.sessionData?.CAMPAIGN_DETAILS?.campaignDetails?.distributionStrat?.resourceDistributionStrategyCode
+    
     const assumptionValues = customProps?.sessionData?.HYPOTHESIS?.Assumptions?.assumptionValues || [];
+    let campaignAssumption={};
```

Biome

[error] This let declares a variable that is only assigned once.

'campaignAssumption' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)